test/unit/events.js

```diff
@@ -66,4 +66,52 @@ describe('custom events', () => {
       done();
     }, 1);
   });
+
+  it('should handle cases correctly', done => {
+    let count = 0;
+    const Comp = reactify(document.registerElement('x-custom-event-4', {
+      prototype: Object.create(HTMLElement.prototype, {
+        trigger: {
+          value() {
+            this.dispatchEvent(new CustomEvent('testThis'));
+          },
+        },
+      }),
+    }), { React, ReactDOM });
+
+    const func = () => ++count;
+
+    // Using both ontest and onTest (case-sensitive) test case-sensitivity.
+    const comp = ReactDOM.render(<Comp ontestThis={func} onTestThis={func} />, window.fixture);
```

Comment:
I think what Trey means is that if we had a component with a prop named `once` (e.g. like underscore has [_.once](http://underscorejs.org/#once)) the current algorithm would interpret that as "onCe", and do a `element.addEventListener("on", …)`.